hyperspy/learn/mva.py

```diff
@@ -166,6 +173,17 @@ def decomposition(self,
 
         """
         to_return = None
+        to_print = [
+            "Decomposition info:",
+            "  normalize_poissonian_noise={}".format(normalize_poissonian_noise),
+            "  algorithm={}".format(algorithm),
+            "  output_dimension={}".format(output_dimension),
+            "  centre={}".format(centre),
+            "  auto_transpose={}".format(auto_transpose),
+            "  polyfit={}".format(polyfit),
+            "  reproject={}".format(reproject)
```

Comment:
I think we could remove these to make it more succinct.

Reply:
I think auto-transpose is important. Agree on the others.

Reply:
That said, `auto_transpose` only affects SVD/fast SVD, so I'll remove it. I initially thought it affected the sklearn estimators but it doesn't.